projects/Allwinner/devices/A20/bootloader/release

```diff
@@ -1,9 +1,11 @@
 # SPDX-License-Identifier: GPL-2.0
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
+SRCDIR="$BUILD/image/system/usr/share/bootloader"
+
 mkdir -p $RELEASE_DIR/3rdparty/bootloader
   if [ -n "$UBOOT_SYSTEM" ]; then
-    cp -a $(get_build_dir $BOOTLOADER)/u-boot-sunxi-with-spl.bin $RELEASE_DIR/3rdparty/bootloader
+    cp -a $SRCDIR/u-boot-sunxi-with-spl.bin $RELEASE_DIR/3rdparty/bootloader
   fi
 
-  cp -a $(get_build_dir linux)/arch/$TARGET_KERNEL_ARCH/boot/dts/sun7i-a20-*.dtb $RELEASE_DIR/3rdparty/bootloader
+  cp -a $SRCDIR/sun7i-a20-*.dtb $RELEASE_DIR/3rdparty/bootloader
```

Comment:
While you are at it, can you remove these device specific files and make one common? Only additional change is to make last line "cp -a $SRCDIR/*.dtb $RELEASE_DIR/3rdparty/bootloader
". There will be a bit more files, but it doesn't really matter much.

Reply:
Done, with the addition of copying only device specific dtb files (`*-${lowercase device name}-*.dtb`)